Car.cpp

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+
+#include "Car.hpp"
+
+Car::Car() {
+    std::cout << "* " << __FUNCTION__ << std::endl;
+}
+
+Car::~Car() {
+    std::cout << "* " << __FUNCTION__ << std::endl;
+}
+
+void Car::turnLeft() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+void Car::turnRight() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+void Car::brake() {
+    std::cout << __FUNCTION__;
+    speed_ = 0;
+    std::cout << " : stopped" << std::endl;
+}
+
+void Car::accelerateTo(int speed) {
+    std::cout << __FUNCTION__;
```

Comment:
What is this?

Reply:
Is was so in the original files. It prints out the name of the method and was used instead of implementing it. So you know when the method, constructor or destructor was called. Of course when there is full implementation those wouldn't be used. It's suitable for testing purposes.